Ochowo/iReporter
# #162465259 get request for intervention

**What does this PR do?**
- Creates an  endpoint to get a specific  record

**Description of the task  to be completed**
- Ensure the endpoint below is working as it should
    `GET/api/v1/interventions/id`

**How should this be tested manually**
   After you clone the repo, cd into it and run `npm install` to install node modules. Make sure you have `nodejs` and express installed. Afterwards, run `npm run start:dev` to initiate the app.
- test the endpoint using postman with `localhost:3000` as the host
- send empty inputs to test validation

**What are the relevant pivotal stories**
- [#162465259](https://www.pivotaltracker.com/story/show/162465259)

**Any background context you want to add?**
- N/A

**Screenshots**
N/A
